web/src/OverviewTableColumns.tsx

```diff
@@ -379,10 +381,12 @@ export function TableNameColumn({ row }: CellProps<RowValues>) {
     row.original.statusLine.buildStatus === ResourceStatus.Unhealthy ||
     row.original.statusLine.runtimeStatus === ResourceStatus.Unhealthy
   const errorClass = hasError ? "has-error" : ""
-  const disabledClass = rowIsDisabled(row) ? "isDisabled" : ""
+  let isDisabled = rowIsDisabled(row)
+  const disabledClass = isDisabled ? "isDisabled" : ""
   return (
     <Name
       className={`${errorClass} ${disabledClass}`}
+      aria-disabled={isDisabled}
```

Comment:
i don't think we want to disable the link to detail view for a disabled resource. is there a reason you added this markup?